llvm/lib/IR/RepoHashCalculator.cpp

```diff
@@ -347,21 +347,22 @@ void HashCalculator::hashGlobalValue(const GlobalValue *V) {
 
   GlobalNumbers.insert(std::make_pair(V, GlobalNumbers.size()));
 
-  if (auto *GV = dyn_cast<GlobalVariable>(V)) {
-    // Accumulate the initial value of global variable .
-    Hash.update(HashKind::TAG_GlobalVariable);
-    if (GV->hasDefinitiveInitializer())
-      hashConstant(GV->getInitializer());
-  }
```

Comment:
Was this code no longer necessary? We still need to include the value of a global’s initializer in the hash, don’t we?

Reply:
> 
> 
> Was this code no longer necessary? 
>
Yes, this code is the redundant code and is removed here.

>
> We still need to include the value of a global’s initializer in the hash, don’t we?
>
Yes, we still need to accumulate the GO's initial hash, which is implemented in the `accumulateGODigest` (line 156 of `RepoDefinition.cpp` file. Here, we only collect the GO's dependencies and contribution. In the recursive `updateDigest` function, the function `accumulateGODigest` is called to accumulate the GO initial digest and his dependencies initial digest.